Common/src/main/java/com/tabnineCommon/binary/requests/login/LoginWithCustomTokenResponse.kt

```diff
@@ -0,0 +1,9 @@
+package com.tabnineCommon.binary.requests.login
+
+import com.google.gson.annotations.SerializedName
+import com.tabnineCommon.binary.BinaryResponse
+
+data class LoginWithCustomTokenResponse(
+    @SerializedName("is_success")
+    var isSuccess: Boolean? = null,
```

Comment:
if this fails, will we have any ability to debug it?

Reply:
currently no.. let's talk about this tomorrow